packages/apps/src/Apps.tsx

```diff
@@ -26,7 +26,8 @@ interface SidebarState {
 
 function WarmUp (): React.ReactElement {
   const { api, isApiReady } = useApi();
-  const fees = useCall<any>(isApiReady ? api.derive.balances?.fees : undefined, []);
+  // const fees = useCall<any>(isApiReady ? api.derive.balances?.fees : undefined, []);
+  const fees = null;
```

Comment:
Is there any impact outside of the explorer for setting this value to null?

Reply:
Doesn't seem so..